packages/content/src/components/content-header/content-header.tsx

```diff
@@ -68,14 +68,18 @@ const ContentHeader = (props: {
         {/* description */}
         <div className="text-lg text-slate-400 pb-2.5 font-primary">{description}</div>
         {/* stars */}
-        {props.showStars && rating && (
-          <>
-            <div className="flex pb-8">
-              <Stars score={rating} />
-              <div className="font-bold px-1">{(rating / 20).toFixed(1)}</div>
-              <div>({ratingsCount} Reviews)</div>
-            </div>
-          </>
+        {props.showStars && (
+          <div className="flex pb-8">
+            {rating ? (
```

Comment:
@AlexONeillTI Think there's still a bit of cleanup we can do here.

1) On line 52, we can initialize `ratingsCount` with a value of `0` instead of leaving it `undefined`. That way it's only updated if we have data from `CourseGroupBySlug`, and if we don't, there's no need for the `or 0 reviews` block.

2) `ratingsCount` is still dependent on `rating` having a truth-y value in order to render, when it should only depend on if `ratingsCount` has a truth-y value. Ex: If 5 learners rate a course as `0` (unlikely but possible); since `rating` is `0`, we would end up rendering `0 Reviews`, which would cause a learner to infer that there are no reviews on the course, when really it's just a bad course.

Reply:
The ratingsCount returns 0 if there are no reviews. So I think we can just return the ratingsCount regardless? Pushed so lmk what you think!